src/types/aid.ts

```diff
@@ -0,0 +1,17 @@
+export interface AID {
+  name: string;
+  prefix: string;
+  salty: Salty;
+}
+
+interface Salty {
```

Comment:
I think these types can also be added to the native signify-ts client rather than in the web app itself! It would be awesome if you could make a pr [here](https://github.com/WebOfTrust/signify-ts/blob/main/src/keri/core/manager.ts#L112) as well to add these!

Reply:
Sure! Thank you for pointing out to this!